ios/gamma/split-view/RNSSplitViewScreenController.swift

```diff
@@ -69,46 +87,144 @@ public class RNSSplitViewScreenController: UIViewController {
 
   // MARK: Layout
 
+  ///
+  /// @brief This method is overridden to extract the value to which we're transitioning
+  /// and attach the DisplayLink to track frame updates on the presentation layer.
+  ///
+  public override func viewWillTransition(
```

Comment:
One thing about this method is that, I believe, UIKit also fires it on device / interface orientation changes. Have you tested that we work fine with orientation changes here?

Reply:
I've checked & seems that orientation change works nicely.